src/docs/asciidoc/user-guide/assertj-core-migration.adoc

```diff
@@ -169,3 +170,24 @@ Find/replace regex:
 ----
 assertFalse\((.*)\); -> assertThat(\1).isFalse();
 ----
+
+[[assertj-migration-using-openrewrite]]
+==== Assertions migration with OpenRewrite
+
+link:https://docs.openrewrite.org/[OpenRewrite], a large-scale automated source code refactoring tool, offers a couple of recipes that assist with the migration to AssertJ:
+
+The link:https://docs.openrewrite.org/recipes/java/testing/hamcrest/migratehamcresttoassertj[Migrate to AssertJ assertions] recipe will:
```

Comment:
Since we're showing this link out of the context of other Hamcrest recipes, I'd though it best to insert Hamcrest here.
```suggestion
The link:https://docs.openrewrite.org/recipes/java/testing/hamcrest/migratehamcresttoassertj[Migrate Hamcrest to AssertJ assertions] recipe will:
```